lte/cloud/go/services/subscriberdb/servicers/subscriberdb_servicer.go

```diff
@@ -207,6 +207,25 @@ func (s *subscriberdbServicer) ListSubscribers(ctx context.Context, req *lte_pro
 	return listRes, nil
 }
 
+func (s *subscriberdbServicer) ListSuciProfiles(ctx context.Context, req *lte_protos.ListSuciProfilesRequest) (*lte_protos.ListSuciProfilesResponse, error) {
+	gateway := protos.GetClientGateway(ctx)
+	if gateway == nil {
+		return nil, status.Errorf(codes.PermissionDenied, "missing gateway identity")
+	}
+	if !gateway.Registered() {
+		return nil, status.Errorf(codes.PermissionDenied, "gateway is not registered")
+	}
+	networkID := gateway.NetworkId
+
+	var suciProtos []*lte_protos.SuciProfile
+	suciProtos, _ = subscriberdb.LoadSuciProtosPage(ctx, networkID)
+
+	listRes := &lte_protos.ListSuciProfilesResponse{
```

Comment:
Same as above: don't name this based on whether it's getting returned, name it based on its contents

Reply:
done